CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ For migration information, you can always have a look at https://liip-drifter.re
 ## Unreleased
 
 ### Added
+- Virtualbox: allow use of the vagrant default synced folders implementation.
```

Comment:
Could you please mention "VirtualBox shared folders" instead of "default synced folders"?